docs/data-types/bitmaps.md

```diff
@@ -21,37 +21,30 @@ Some examples of bitmap use cases include:
 
 * `SETBIT` sets a bit at the provided offset to 0 or 1.
 * `GETBIT` returns the value of a bit at a given offset.
-* `BITOP` lets you perform bitwise operations against one or more strings.
 
 See the [complete list of bitmap commands](https://redis.io/commands/?group=bitmap).
 
 
-## Examples
+## Example
 
-Suppose you have 1000 sensors deployed in the field, labeled 0-999.
-You want to quickly determine whether a given sensor has pinged the server within the hour. 
+Suppose you have 1000 cyclists racing through the country-side, labeled 0-999.
+You want to quickly determine whether a given rider has pinged a tracking server within the hour. 
```

Comment:
Sorry, but I think connected sensors are more common than connected cyclists. Or maybe it's a censor on each bicycle? :)